x-pack/plugins/lists/README.md

```diff
@@ -0,0 +1,254 @@
+README.md for developers working on the backend lists on how to get started
```

Comment:
Love docs! Would it be worth adding info on the spaces stuff with exception lists?